### PR TITLE
Create Rule “less-is-more”

### PR DESCRIPTION
**Pull request by 2fernandez**
A new standard to simplify our approach on all things design, marketing and content.

### CODE REVIEW

File: rules/less-is-more/rule.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+## Keep it simple
+
+
By keeping things simple we reduce complexity and avoid the cognitive overload required for a user to make an informed decision.
 We want to keep things easily digestible to as broad an audience as possible. Again mainly focus the user on the completion of a task. 
+
+Using simple descriptive language or a minimalist design approach can often demystify complicated topics and make information easily digestible so that users can make decisions with ease.
+
+![Figure: ❌ Bad example - Responding with no clear answer](responding-bad-example.jpg)
```
Comment (tiagov8) [inline rules/less-is-more/rule.md:43]: Hey @2fernandez 

See how captions are done https://ssw.com.au/rules/rule/#7-captions
You won't need to add the emoji if you use the right markdown syntax

1. Please fix

File: rules/less-is-more/rule.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+![Figure: ❌ Bad example - video text over html text](text-over-text.jpg)
+
+![Figure: ✅ Good example (https://www.sanleonardo.it) - use of background video with html text](san-leonardo-website.jpg)
+
+Defining user empathy at various stages will help you develop a deeper understanding of the user's needs and ultimately help your products or assets to be used over and over again.
+
+Developing empathy maps for a user might be a handy tool to use when you are trying to empathise with the user.
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/QwF9a56WFWA" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
```
Comment (tiagov8) [inline rules/less-is-more/rule.md:61]: Hey @2fernandez 

See how videos are added in rules: https://ssw.com.au/rules/rule/#8-videos 

You don't need the whole <iframe> if you use the right markdown syntax

1. Please fix

File: rules/less-is-more/rule.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+❌ It does not mean we Include or develop a feature in a design or product just because we have the technical ability to do so
+
+✅ When we talk about "Less is More", we are talking about simplifying the way we present ideas to focus the user on making decisions about a task in the most effective way possible.
+
+Here are a few helpful ways we can employ a “Less is more” approach:
+
+## Leave it out:
+
+"Just because we can does not mean we should"
```
Comment (tiagov8) [inline rules/less-is-more/rule.md:27]: 1. Please make this a quote as per https://ssw.com.au/rules/rule/#1-headings-paragraphs-and-blockquotes

File: rules/less-is-more/rule.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+uri: less-is-more
+authors:
+  - title: Josep Fernandez
+    url: https://www.ssw.com.au/people/joseph-fernandez
+created: 2023-02-22T04:20:12.183Z
+guid: 9d8ce87b-f835-48a2-b60c-5549466da209
+---
+A popular concept in product development and interphase design is the idea of “Less is more”. 
But what do we actually mean by less is more? 
+
```
Comment (tiagov8) [inline rules/less-is-more/rule.md:12]: This rule is missing the markdown tag that separates the blurb view from full content:

`<!--endintro-->`

I would add it here

1. Please action

File: rules/less-is-more/rule.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
+---
+type: rule
+title: Do you know that Less is more ?
+uri: less-is-more
+authors:
+  - title: Josep Fernandez
```
Comment (tiagov8) [inline rules/less-is-more/rule.md:6]: Missing a letter in your name

1. Please fix

File: rules/less-is-more/rule.md [hunk 6]
```diff
@@ -0,0 +1,61 @@
+---
+type: rule
+title: Do you know that Less is more ?
```
Comment (tiagov8) [inline rules/less-is-more/rule.md:3]: 1. Remove the space before "?"

File: rules/less-is-more/rule.md [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+
+❌ It doesn’t mean we make decisions based on our own personal biases or perceptions of what we think is the easiest solution for the user
+
+❌ It does not mean we Include or develop a feature in a design or product just because we have the technical ability to do so
+
+✅ When we talk about "Less is More", we are talking about simplifying the way we present ideas to focus the user on making decisions about a task in the most effective way possible.
+
+Here are a few helpful ways we can employ a “Less is more” approach:
+
+## Leave it out:
```
Comment (tiagov8) [inline rules/less-is-more/rule.md:25]: 1. Remove colon from this heading

File: rules/less-is-more/rule.md [hunk 8]
```diff
@@ -0,0 +1,61 @@
...

+
+The word empathise means that you start to subjectively put yourself in the user’s shoes and try to experience what they are feeling. 
Try to forecast how your design choices will impact other productions that might use it.
+
+Don’t forget! Most users will only invest time if it benefits them and if it is easy to process a task with minimum frustration. A large portion of users will disengage with your content if you make it harder for them to digest.
+
+![Figure: ❌ Bad example - video text over html text](text-over-text.jpg)
+
+![Figure: ✅ Good example (https://www.sanleonardo.it) - use of background video with html text](san-leonardo-website.jpg)
```
Comment (tiagov8) [inline rules/less-is-more/rule.md:55]: I don't think Adam will be happy having a bad example as SSW and a good example as someone else

1. Please find a good example within SSW sites

File: rules/less-is-more/rule.md (hunk 1 above)
Comment (tiagov8) [inline rules/less-is-more/rule.md:43]: 1. Can you remove/hide people's names from the image?

File: rules/less-is-more/rule.md [hunk 9]
```diff
@@ -0,0 +1,61 @@
...
+Here are a few helpful ways we can employ a “Less is more” approach:
+
+## Leave it out:
+
+"Just because we can does not mean we should"
+
+Including a design element, feature or product enhancement, because we have the technical ability to do so with no impact on the user experience. It could overcomplicate a simple task, cause cognitive overload and ultimately distract an audience from processing through a task with less friction. 
It’s Equally as important to leave out a good idea than it is to include one, especially if it doesn’t help the user achieve their goals.
+
+![Figure: ❌ Bad example - overbranded background](microsoftteams-image-withlogos.png "Figure: Bad")
```
Comment (tiagov8) [inline rules/less-is-more/rule.md:31]: Would these images of the background be better without all the audience heads? Right now both images look too cluttered...
I think it should be zoomed in to the area that matters.

1. Please update if agreed

File: rules/less-is-more/rule.md [hunk 10]
```diff
@@ -0,0 +1,61 @@
...
+A popular concept in product development and interphase design is the idea of “Less is more”. 
But what do we actually mean by less is more? 
+
+Let’s start off by defining what it doesn’t mean:
+
+❌ It doesn’t mean that you work less on a project and be able to deliver more
+
+❌ It doesn’t mean we make decisions based on our own personal biases or perceptions of what we think is the easiest solution for the user
+
+❌ It does not mean we Include or develop a feature in a design or product just because we have the technical ability to do so
```
Comment (tiagov8) [inline rules/less-is-more/rule.md:19]: 1. Fix capital "Include"

File: rules/less-is-more/rule.md (hunk 8 above)
Comment (2fernandez) [reply rules/less-is-more/rule.md]: Done!

File: rules/less-is-more/rule.md (hunk 9 above)
Comment (2fernandez) [reply rules/less-is-more/rule.md]: Done!

**Review by tiagov8 — APPROVED** [review]

**Pull request merged**